### ISSUE TITLE
Pain tile is projected across every tile but not visable to players

### ISSUE THREAD

**Issue by Jeff-Dome** [WW2-SS13/WW13#639]
it's across every tile on the map

**Comment by KanohaShinobi** [comment]
yeah, it's supposed to be like that. It's not visible to anyone. It's the same on civ13, it's there so you can feel pain on that tile I'm pretty sure. Or something connected to pain and is supposed to be there.

**Comment by Jeff-Dome** [comment]
that's kind of a bad way to do it as every tile is procressing pain

**Comment by Jeff-Dome** [comment]
if it has a client in pain
